### PR TITLE
Listener + discovery multi-contract fix

### PR DESCRIPTION
**Pull request by nick**
- Fix up some READMEs
- Listener for marketplace should default to v0
- GraphQL fix for discovery result listing IDs.
- Add listener and discovery to `@origin/services`

### CODE REVIEW

File: packages/services/index.js [hunk 1]
```diff
@@ -205,6 +202,67 @@ const startRelayer = () =>
...
+    const cwd = path.resolve(__dirname, '../../infra/discovery')
+
+    const spawnedListener = spawn(
+      `node`,
+      [
+        'src/listener/listener.js',
+        '--network=localhost',
+        '--marketplace',
+        '--identity',
+        '--elasticsearch'
```
Comment (franckc) [inline packages/services/index.js]: let's also add --growth (it captures events in the Db for Origin Rewards).

File: infra/discovery/src/apollo/README.md [hunk 2]
```diff
@@ -1,38 +1,36 @@
...
-```
-
-The server should start and you can now point your browser to http://localhost:4000/graphql to access the GraphQL playground.
+### Lerna
+
+#### Prerequisite
+
+- Setup Postgres locally.
+- Run migration scripts to create the schema
+  `lerna run migrate --scope origin-discovery`
```
Comment (franckc) [inline infra/discovery/src/apollo/README.md]: I think it is --scope @origin/discovery
Also the identity, growth and bridge migrations need to get run as well
lerna run migrate --scope @origin/identity
lerna run migrate --scope @origin/growth
lerna run migrate --scope @origin/bridge

File: packages/services/index.js (hunk 1 above)
Comment (nick) [reply packages/services/index.js]: ok, will add in a separate PR

File: infra/discovery/src/apollo/README.md (hunk 2 above)
Comment (nick) [reply infra/discovery/src/apollo/README.md]: Will do another pass of READMEs... I think a lot of them are out of date

**Pull request merged**